views/layouts/HeroLayout/index.jsx

```diff
@@ -1,29 +1,83 @@
 import React from 'react'
 import { Outlet } from 'react-router-dom'
-import { Box } from '@mui/material'
+import { Box, Grid } from '@mui/material'
 
 import HeroCard from '../../components/HeroCard'
 import HeroFooter from '../../components/HeroFooter'
-import './HeroLayout.css'
 
 const HeroLayout = () => {
   return (
-    <Box className="HeroLayout_container">
-      <Box className="HeroLayout_main">
-        <HeroCard />
-
-        <Box
+    <Box sx={{ height: '100vh' }}>
```

Comment:
I've got personal gripes with component-based grid systems but I guess this is the Material Way.